service/common/src/main/java/org/apache/polaris/service/admin/PolarisServiceImpl.java

```diff
@@ -107,12 +114,7 @@ private PolarisAdminService newAdminService(
     PolarisMetaStoreManager metaStoreManager =
         metaStoreManagerFactory.getOrCreateMetaStoreManager(realmContext);
     return new PolarisAdminService(
```

Comment:
Question: are we going to use DI for the class `PolarisAdminService` later as well?

Reply:
Probably. There is a huge overlap between `PolarisServiceImpl` (which is a request-scoped CDI bean) and `PolarisAdminService`, which is just a regular Pojo.